hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/GrpcOmTransport.java

```diff
@@ -113,7 +115,7 @@ public GrpcOmTransport(ConfigurationSource conf,
   }
 
   public void start() throws IOException {
-    if (!isRunning.compareAndSet(false, true)) {
+    if (isRunning.get()) {
```

Comment:
We should keep `compareAndSet` for atomic operation, but move this after the address lookup.

Reply:
Alright, looks good thanks.  I put back previous atomic `compareAndSet` for `isRunning` boolean and placed it _after_ the exception generating address lookup call.  Changes reflected in latest commit update.